WordPress/gutenberg
# Fix HStack and VStack alignment prop

## What?

While working on #47827 I've noticed a bug in VStack component. The alignment prop was not working as expected. It turns out that there are several bugs in the alignment prop of both VStack and HStack, This PR fixes it.

## Testing Instructions

1- Open the storybook of both VStack and HStack
2- Check that all the alignment options behave as expected.
